### PR TITLE
Remove unwanted  FABs

### LINKED ISSUE #3186: Remove FABs (5pts)
- Remove Search FAB
- Remove Reset FAB and add Clear Filters button on top that does the same 

fabs out

### PR DESCRIPTION
**Pull request by grafitto**
fixes #3186 

PR checklist:
- [ ] Update READ.me ?
- [ ] Update API documentation ?

QA checklist:
- [ ] Smoke test the functionality described in the issue
- [ ] Test for side effects
- [ ] UI responsiveness
- [ ] Cross browser testing
- [ ] Code review

### CODE REVIEW

File: nightmare/helpers/selectors.js [hunk 1]
```diff
@@ -161,11 +161,11 @@ export default {
     documentAfterLoadMore:
       '#app > div.content > div > div > div > main > div > div > div.item-group > div:nth-child(31)',
     superVillianType:
-      '#app > div.content > div > div > div > aside.side-panel.library-filters > div.sidepanel-body > div.documentTypes-selector.nested-selector > ul > li:nth-child(2) > label',
+      '#app > div.content > div > div > div > aside.side-panel.library-filters > div.sidepanel-body > div > ul > li:nth-child(2) > label', //.documentTypes-selector.nested-selector
```
Comment (fnocetti) [inline nightmare/helpers/selectors.js:164]: Please remove comment

File: nightmare/helpers/selectors.js [hunk 2]
```diff
@@ -285,7 +285,8 @@ export default {
     uploadsNavButton:
       '#app > div.content > header > ul > li.menuActions > ul > li:nth-child(2) > a',
     libraryNavButton:
-      '#app > div.content > header > ul > li.menuActions > ul > li:nth-child(1) > a',
+      '#app > div.content > header > ul > li.menuActions > ul.menuNav-list > li:nth-child(1) > a',
+    // '#app > div.content > header > ul > li.menuActions > ul > li:nth-child(1) > a',
```
Comment (fnocetti) [inline nightmare/helpers/selectors.js:289]: Remove comment here as well

**Review by fnocetti — CHANGES_REQUESTED** [review]
In general it LGTM. Please check the comments.

**Comment by fnocetti** [comment]
The testing went just OK. I'll wait for the changes to merge.

File: nightmare/helpers/selectors.js (hunk 2 above)
Comment (grafitto) [reply nightmare/helpers/selectors.js]: Done

File: app/react/App/scss/layout/_sidepanel.scss [hunk 3]
```diff
@@ -300,6 +300,35 @@ $c-sidebar: $c-white;
...
+        cursor: pointer;
+        border-radius: 2px;
+      }
+
+      @media(min-width: 1020px) {
+        .push-right {
+          margin-left: 30px;
+        }
+      }
+    }
```
Comment (RafaPolit) [inline app/react/App/scss/layout/_sidepanel.scss:331]: Please indent this correctly.

**Comment by RafaPolit** [comment]
I am sorry if this comes late, but this design is not useful without having the filter title being a sticky.  In order to reset filters on a very long list you would need to scroll back up to the title.  The precise reason the reset button was made a fab.

@simonfossom was this the intended behavior?

**Review by RafaPolit — CHANGES_REQUESTED** [review]
I have commented about the scrolling issue, please do not merge this without confirming design problems.

**Comment by fnocetti** [comment]
@RafaPolit noted. That's a fair point.

**Comment by simonfossom** [comment]
> I am sorry if this comes late, but this design is not useful without having the filter title being a sticky. In order to reset filters on a very long list you would need to scroll back up to the title. The precise reason the reset button was made a fab.
> 
> @simonfossom was this the intended behavior?

I understand the concern but how many filters should there be before quick scroll to the top is a problem?
Another argument is that clearing filters is secondary action to selecting filters. With that rational all filters should always be in view. 
Yet another argument would be that already short hight of filter panel would be even shorter (which is also one of the reasons why FABs are a problem.).

I don't know. What's the reason for having it always visible? I couldn't find any instance where I needed to scroll more than once to reach the top of filters.

**Comment by nwyu** [comment]
Fair point raised by @RafaPolit, but @grafitto let's proceed as it is designed. Please ensure, as well, that the Clear Filters button is next to the x on mobile.

**Comment by fnocetti** [comment]
@nwyu ok, this is noted. I'll finish the QA and merge this as it is
cc @RafaPolit

File: app/react/App/scss/layout/_sidepanel.scss (hunk 3 above)
Comment (grafitto) [reply app/react/App/scss/layout/_sidepanel.scss]: Done

**Comment by grafitto** [comment]
> Fair point raised by @RafaPolit, but @grafitto let's proceed as it is designed. Please ensure, as well, that the Clear Filters button is next to the x on mobile.

Yes it is.

**Pull request merged**